docsrc/allblocks.nim

```diff
@@ -17,7 +17,7 @@ template nbCodeBlock(name:string) =
   nbToc.output.add "1. <a href=\"#" & anchorName & "\">" & name & "</a>\n"
 
 nbText: """
-> This nimib document provides a brief description and example for 
+> This nimib document provides a brief description and example for
```

Comment:
I'm sorry, the diff contains a lot of line changes because of some automatic tool in my Vim editor to remove whitespaces at the end of lines.